### DISCUSSION TITLE
Will Basic Order Data Also Require PII Approval and RDT Token?

### DISCUSSION THREAD

**Discussion by jimmy-ross** [amzn/selling-partner-api-models#3637] [General]
Hi, 

Im trying to get a clear answer on this announcement that is supposed to take effect on August 31: 
[https://developer-docs.amazon.com/sp-api/changelog/important-notice-discontinued-support-lwa-tokens-can-no-longer-retrieve-pii-in-selling-partner-api](https://developer-docs.amazon.com/sp-api/changelog/important-notice-discontinued-support-lwa-tokens-can-no-longer-retrieve-pii-in-selling-partner-api)

I understand the PII Approval and the use of RDT Tokens will be required for retrieving PII data.

- Will PII Approval and the RDT token process be required even for the most basic order data that is currently available via get order, get orders, get order items? So parts that are not PII?
- So does this change mean that anyone currently using get orders for basic order reporting has to get PII approval?
- Or does it mean PII Approval and RDT Token is needed for operations **within** the orders endpoint that include actual PII:  for example retrieving the Buyer Info

This distinction is really important to me (and I assume others) as it will help clarify if going through the PII Approval process is required for basic order tracking use cases.

**Comment by jamesquinton** [comment]
I submitted a support ticket regarding this and did not receive a clear answer. Is anyone from the SP-API team able to clarify this? So far I have not seen any concrete response to any questions regarding this, or the application process for the RDT restricted roles. @rugved1991 or @jevoniuk we would really appreciate some more info here - as @jimmy-ross-xapix says, this is very important for many of your API users.

**Comment by jamesquinton** [comment]
@jimmy-ross-xapix I had a call with a representative from the API team yesterday. They assured me that we will be able to access these endpoints and return non-PII data without RDT tokens after the change on 31st August. The announcement is aimed at developers who have migrated from the MWS API where it was possible to return PII using LWA tokens. 

I did ask that they publish a clarification as this was not clear from the original announcement, and has taken quite a lot of our time to investigate and chase up responses.

**Comment by jasonhebert** [comment]
I also got a clarification response on our ticket stating access to basic order data not containing the PII will still be provided under the Inventory and Order Tracking role:

 "I would like to explain to you that in this case we have (as an example ) the operation getOrders under the Inventory and Order Tracking role and also the same operation, getOrders with RDT under Direct to Consumer Shipping (Restricted) role, In this case the main reason is because in the first case you wont be getting PII information, will be able to retrieve all the information you need except the PII."
